cloud/shared/bin/run

```diff
@@ -40,20 +41,24 @@ function fetch_json_val() {
     | python3 -c "import sys, json; print(json.load(sys.stdin)${2})"
 }
 
-# Reassign tag to version so we don't overwrite tag
-version=${tag}
-# If version is "latest" get the latest release number (eg. v1.26.0)
-if [[ ${version} == "latest" ]]; then
-  version=$(fetch_json_val \
-    "https://api.github.com/repos/civiform/civiform/releases/latest" \
-    "['tag_name']")
-fi
+commit_sha=""
 
-# Get the commit sha for the commit at the tip of the CiviForm version being deployed
-tag_url=$(fetch_json_val \
-  "https://api.github.com/repos/civiform/civiform/git/refs/tags/${version}" \
-  "['object']['url']")
-commit_sha=$(fetch_json_val ${tag_url} "['object']['sha']")
+if [[ ${tag} == "latest" ]]; then
```

Comment:
**nitpick (blocking):** quote variables here and L49 https://google.github.io/styleguide/shellguide.html#quoting
```suggestion
if [[ "${tag}" == "latest" ]]; then
```

Reply:
done